cli/dependabot-cmd.ts

```
@@ -22,14 +111,66 @@ async function fetchDependabotPRs(): Promise<PR[]> {
+      await new Promise((resolve) => setTimeout(resolve, pollIntervalMs));
+    }
+  }
+
+  if (remainingPRs.size > 0) {
+    console.log(`\n⚠️  Timeout: ${remainingPRs.size} PR(s) did not reach SUCCESS state: ${Array.from(remainingPRs).join(", ")}`);
+  } else {
+    console.log(`\n✓ All PRs reached a final state`);
+  }
+}
+
+async function applyPR(pr: PR, rebase: boolean, useAutoMerge: boolean = false): Promise<void> {
```

ESLint

[error] Type boolean trivially inferred from a boolean literal, remove type annotation.

(@typescript-eslint/no-inferrable-types)